source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -55,25 +55,34 @@ def filter_journeys(responses, instance, request):
     Filter by side effect the list of pb responses's journeys
 
     """
-    is_debug = request.get('debug')
+    is_debug = request.get('debug', False)
+    min_bike = request['_min_bike']
```

Comment:
just an OCD's comment:  :smile: 

min_bike = request.get('_min_bike') ?

Reply:
I would agree with XL only if you set a default value ;)